### ISSUE TITLE
Plugins/Themes: Install or upload? Messaging should be consistent

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#33062]
<img width="1031" alt="Screen Shot 2019-05-15 at 12 15 13 PM" src="https://user-images.githubusercontent.com/2124984/57791700-57a09c80-770b-11e9-87fe-b174ad4c1684.png">

We use two different terms ("Upload" and "install) for the same action on the same page, and throughout the Plugins section.

* Can we make these consistent and use one or the other going forward? 
* Which term is less jargon-y and easier to understand? How does this compare to other services that offer third-party plugin integration?
* We should also consider using the same terminology for uploading/installing third-party themes.

Found during this walkthrough by @ianstewart https://cloudup.com/cTAV7Njdy1z

**Comment by sixhours** [comment]
Another instance of the same discrepancy when uploading/installing a plugin:

<img width="583" alt="Screen Shot 2019-05-15 at 1 10 23 PM" src="https://user-images.githubusercontent.com/2124984/57794969-dc42e900-7712-11e9-9543-ce1349175e5d.png">

**Comment by danieldudzic** [comment]
To me, Install refers to a scenario where the user enables an existing module (and installs it on their site).

Upload, on the other hand, implies that it's a custom plugin/theme that I am uploading from my machine.

Perhaps it would make sense to pair the term "upload" with "custom" in appropriate places.

Trying to put myself in the shoes of a user, I think I'd interpret the notification from the screenshot, in the following way:

> If I upgrade to Business plan, I'll be able to Install plugins that are listed below

Having the ability to upload plugins wouldn't be clear to me.

So, I'd leave "Upload Plugin" on the upload button as it is and I'd reword the notification to mention both Install and Upload.

For example:

> Upgrade to the Business plan to install plugins from our library/repository/collection and upload custom plugins

**Comment by danieldudzic** [comment]
> Another instance of the same discrepancy when uploading/installing a plugin:
> 
> <img alt="Screen Shot 2019-05-15 at 1 10 23 PM" width="583" src="https://user-images.githubusercontent.com/2124984/57794969-dc42e900-7712-11e9-9543-ce1349175e5d.png">

This should say 

> Uploading your plugin
 or 
> Uploading your custom plugin

**Comment by danieldudzic** [comment]
Interesting, the Plugin Upload plugin actually does make a distinction and should display a message accordingly:

![wp-calypso](https://user-images.githubusercontent.com/905781/59311639-24482380-8c5f-11e9-9ebd-965f22c34209.jpg)

**Comment by danieldudzic** [comment]
Wix uses Add to Site

![e39e646d-8ed3-4d95-831f-f0fe62dfd882](https://user-images.githubusercontent.com/905781/59312169-f5cb4800-8c60-11e9-8042-7988a292d0bd.png)

Shopify uses Add app

![Shopify](https://user-images.githubusercontent.com/905781/59312549-6888f300-8c62-11e9-9ca0-a572ec1ccaab.jpg)

**Comment by drw158** [comment]
Look through WP Admin. Looks like there are a few different terms they use:

**Add** plugin: used to describe adding a plugin either by **uploading** or **installing** plugins using the plugin browser.

Once a plugin has been either **uploaded** or **installed**, you can **activate** it.

**Activate**: enable a plugin that has already been uploaded or installed

**Comment by sixhours** [comment]
I think "Add Plugin" is clearer than Upload, and less jargon-y. So the upgrade nudge would read:

> Upgrade to the Business plan to add plugins.

The button would read:

> Add Plugin

The heading on that secondary page would also read "Add Plugin" since that's the whole action being taken on that page. The status bar would tell us what part of the process is occurring ("Uploading your plugin" followed by "Installing your plugin")

**Comment by sixhours** [comment]
@obi2020 Do you have thoughts on the wording for installing/uploading/adding plugins? Lots to consider in the research above. :)

**Comment by obi2020** [comment]
Hmmm, I'm trying to think outside of my bias, which is toward _install_. There's a fine line between removing jargon and dumbing down (or sterilizing) and I don't always see that line very clearly.

It seems like a safe bet is to standardize to _add_, but I'm not entirely clear I like that, TBH. As I develop some perspective on what I would like be the brand voice within the product I sort like the idea of making it easy for user to feel like they are active participants in building and "install" feels better for that. I could be way off base re user motivation, though.

Brand & editorial folks, what say you? @benhuberman @michelleweber @ Greg Fox (whom is not on github, i guess?)

**Comment by michelleweber** [comment]
I lean toward Add, because Install for me creates the impression that more work is required on my part than is actually required. But I don't feel super duper strongly about that.

**Comment by obi2020** [comment]
Greg (whom I'm right now recruiting from Slack) likes _add_, too. **I think the Adds have it, then!**

**Comment by obi2020** [comment]
I was ready to acquiesce to _Add_, but Greg and I have been chatting on Slack and he helped me uncover one of my hesitations about the word:

It just feels too light in a way. Like adding something to a cart or a wishlist, as Greg said. Install is much more descriptive of the user taking action to change how their site works. I think if we make it easy to take next steps (activate and/or adjust settings) after installation it can still work. 

@sixhours Sorry if I'm muddying the waters, but I think changing this wording is actually kind of a big deal re: brand voice in the product

**Comment by obi2020** [comment]
From Greg:

1. If we say add anywhere else for other plug in actions (add plug in to your cart, for example) we’ll create confusion down the road.

2. If we’re using “add” for unrelated actions (add a blog or add e-commerce capabilities), do we invite confusion?

3. If we use _install_ it’s a very specific action for plug ins. It’s not as simple as _add_ , but it’s probably a lot clearer because I feel like we’ll probably say _add_ quite a bit for other things, and I want to make sure we’re not creating confusion down the road.

**Comment by sixhours** [comment]
> Sorry if I'm muddying the waters, but I think changing this wording is actually kind of a big deal re: brand voice in the product

No, this is good feedback, thank you! Maybe we do what Danny suggested, and leave the current terms alone, but make it clearer in the upgrade nudge that you can both install or upload custom plugins:

> So, I'd leave "Upload Plugin" on the upload button as it is and I'd reword the notification to mention both Install and Upload.

Or maybe the current terminology is OK as-is and I'm overthinking it? <--- entirely possible. :D

**Comment by JavonDavis** [comment]
> Or maybe the current terminology is OK as-is and I'm overthinking it?

Personally I'm thinking the current terminology is ok, unless there's any strong opinions for changing this I think we should close this one for now, @sixhours WDYT?

**Comment by sixhours** [comment]
That's fine with me. I do wonder if some quick research into live chats to see how often this comes up (if at all) would be a good final indicator as to whether this is something that actually needs to be "fixed". I'm guessing it doesn't, but it can't hurt to see if the research backs that up.

**Comment by jaclync** [comment]
update: I'm updating "upload" to "install" following @obi2020's suggestions
